db/agents.yml

```diff
@@ -783,7 +783,7 @@ agents:
     examples:
       - PodcastAddict/v2 - Dalvik/2.1.0 (Linux; U; Android 8.0.0; SM-G950U Build/R16NW)
       - Podcast Addict - Dalvik/1.6.0 (Linux; U; Android 4.4.4; XT1080 Build/SU6-7.7)
-  - regex: '^(com\.evolve\.)?podcast\/.+(CFNetwork|iOS)'
+  - regex: '^(com\.evolve\.podcast|podcast|ThePodcastApp)\/.+(CFNetwork|iOS)'
```

Comment:
This is a very eager matcher
It matches, `podcast/Spotify iOS`, and almost anything that starts with `podcast/`

Reply:
Though I guess it's been doing that all along, just in a different way

Reply:
Hmm... good point.  Let me break out `podcast/[0-9]+ CFNetwork` into something more specific.